### ISSUE TITLE
[Feature Request] Ability to archive Tags and Organisations

### ISSUE THREAD

**Issue by martini1992** [eikek/docspell#2099]
I believe it would be useful to archive Tags and Organisations which are no longer being actively dealt with  for instances old places of work or types of document which you no longer receive, I still want the tags/organisations to exist for old documents metadata, but I don't want them appearing in the menu as assignable they just clog up the menu and push useful entires further down.

**Comment by eikek** [comment]
Sorry for the late reply, this got buried... So the organization already has this feature (kind of). You can mark any organization as "disabled". This causes it to be not considered in the suggestions. They are still in the menu, because (at least in my case) you could be adding older documents. I didn't feel well to completely remove them from being assignable in the ui. But I can see the advantage! So perhaps we can find some way to allow them being included. It could be a ui feature and a setting to include/exclude these options in the menu. wdyt?

For tags: this will be a bit harder to do the same way, because they are not prepared for that. To make the implementation simpler, it might be possible to hide tags with a certain word/pattern from the list in the ui. Then you could rename old tags to `Archived/Something` and the ui could hide everything starting with `Archived/` for example. OTOH it might be good to have this done properly so that these tags are also not considered during auto-tagging.

**Comment by madduck** [comment]
Maybe what you want is a flag for each tag that controls whether the tag can be assigned. If it cannot, then auto-tagging can't consider it, and it also makes no sense for it to show up in suggestions. The UI could provide a filter like "hide empty", and/or display unassignable tags in a gray colour or something like that?

**Comment by eikek** [comment]
> Maybe what you want is a flag for each tag that controls whether the tag can be assigned. If it cannot, then auto-tagging can't consider it, and it also makes no sense for it to show up in suggestions. The UI could provide a filter like "hide empty", and/or display unassignable tags in a gray colour or something like that?

Yes exactly. That's very much how it works for organization/person. It can be done analog for tags - basically what I meant with "doing properly" 😄 I also think now it's better to do it like this vs. being a ui feature only.

**Comment by martini1992** [comment]
Yeah, I think this sounds like the best solution.
